vision.md

```diff
@@ -0,0 +1,115 @@
+
+
+ ## What is our vision?
+
+The aim of creating thin-edge.io is to provide a IoT edge device framework for IoT project teams which makes it easy to enable resource constrained devices for IoT. Unlike other solutions, we are not just another single-purpose agent but a flexible platform with re-usable components without any vendor lock-in, focused on adressing the needs of both IT and OT users. 
```

Comment:
I would not use passive voice here -> "The aim of thin-edge.io is to provide"...

I also think that saying "Unlike other solutions" is not that interesting to potential readers. What other solutions? Why is not being a single-purpose agent a good thing?

What about: "thin-edge.io is a flexible platform focused on bridging the gap between IT and OT users. It achieves this using re-usable components and without any vendor lock-in!"

Reply:
@TheNeikos "Why is not being a single-purpose agent a good thing?" 
Answer: this is one of the reasons we started thin-edge.io, because we have collected evidence that:
- single purpose agents are hard to maintain and do not offer the flexibility needed for iot solutions
- single purpose agents are often bound to a specific device 
- single purpose agents have usually only few people who can develop them , and those embedded devlopers are hard to find

Reply:
thin-edge.io is a flexible platform focused on bridging the gap between IT and OT users. It achieves this by using reusable components and without any vendor lock-in. - that's what we agreed to replace it with